plugins/org.jkiss.dbeaver.erd.ui/OSGI-INF/l10n/bundle_fr.properties

```diff
@@ -0,0 +1,37 @@
+editor.erd.name=Diagramme ER
+editor.erd.description=Diagramme entit\u00e9-relation
+menu.create.new.diagram=Cr\u00e9er un nouveau diagrame ERD
+pref.page.name.erd.editor=Editeur de diagramme
+page.org.jkiss.dbeaver.preferences.erd.general.name=Param\u00e8tres ERD
+wizard.erd.name=Diagramme ERD
```

Comment:
```suggestion
wizard.erd.name=Diagramme ER
```

Reply:
ERD = Entity Relationship Diagram

Reply:
Update to `Diagramme entité-association`